tests/cross/src/test/scala/tests/hover/HoverDefnSuite.scala

```diff
@@ -180,10 +202,10 @@ class HoverDefnSuite extends BaseHoverSuite {
       |  }
       |}
       |""".stripMargin,
-    """```scala
-      |head: Int
-      |```
-      |""".stripMargin
+    "head: Int".hover,
```

Comment:
this is a change also for scala2, something was also fixed for scala2 here?

Reply:
It's just a simpler way of adding the additional quotes. It wasn't used in this test case, but doesn't change anything.